.env

```diff
@@ -52,6 +52,8 @@ DISCORD_SEND_TEAM_KILL_UPDATES=yes
 VOTE_MAP_INSTRUCTIONS=To vote write the map number in the chat
 # The steam API is used to display country flags and VAC bans. Get a key there: https://steamcommunity.com/dev/apikey
 STEAM_API_KEY=
+# Automatically ban a player if they have been VAC banned within the specified amount of days.
```

Comment:
I'd rephrase that to somthing like: "if they had a VAC ban less than N days ago" to avoid any ambiguty.
Also make 0 the default value and specify in the command that 0 means "disabled"